### PR TITLE
[O2B-821] Make all public routes private 

### PR DESCRIPTION
**Pull request by graduta**
#### I have a JIRA ticket
- [x] branch and/or PR name(s) include(s) JIRA ID
- [x] issue has "Fix version" assigned
- [x] issue "Status" is set to "In review"
- [x] PR labels are selected

Notable changes for users:
- HTTP API routes will become private, thus needing a token to be accessible

Notable changes for developers:
-

Changes made to the database:
-

### CODE REVIEW

File: lib/server/routers/runs.router.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ module.exports = {
     children: [
         {
             path: '/runs',
-            args: { public: true },
```
Comment (martinboulais) [inline lib/server/routers/runs.router.js:20]: Probably best is to simply put `args: {public: true}` on top and remove the sub-ones no?

**Review by martinboulais — CHANGES_REQUESTED** [review]

File: lib/server/routers/runs.router.js (hunk 1 above)
Comment (graduta) [reply lib/server/routers/runs.router.js:20]: You mean `public: false` I guess?
This is the default behaviour of the server in any way, but I let it as false to be explicit

Comment (martinboulais) [reply lib/server/routers/runs.router.js:20]: Yes public false.
TBH I would rather delete it from everywhere if it's the default, to me it looks like dead code.
But my comment was: removing it from children and putting it only at the top

Comment (graduta) [reply lib/server/routers/runs.router.js:20]: I was not sure if you want to keep the convention as it was from the others.
Done, I removed all and let the default `visible` to be applied by the server.

**Review by martinboulais — DISMISSED** [review]

**Review by martinboulais — APPROVED** [review]

**Pull request merged**